README.md

```diff
@@ -33,16 +33,16 @@ Usage:
 
 ```
 import {Inject} from 'angular2/core';
-import {DEFAULT_FIREBASE_REF} from 'angularfire2';
+import {FirebaseRef} from 'angularfire2';
 ...
 class MyComponent {
-  constructor(@Inject(DEFAULT_FIREBASE_REF) ref:Firebase) {
+  constructor(@Inject(FirebaseRef) ref:Firebase) {
```

Comment:
definitely worth considering the ergonomics here - `@Inject(FirebaseRef) ref:Firebase` vs `ref:Firebase` vs `ref:FirebaseRef`